synapse/tools/axon/dump.py

```diff
@@ -77,7 +78,8 @@ async def dumpBlobs(opts, outp):
                     sha2hex = s_common.ehex(sha256)
                     outp.printf(f'Dumping blob {sha2hex} (size={size}, offs={offs})')
 
-                    with s_common.tmpfile(opts.outdir, prefix='axon.dump') as (tmpf, tmp_path):
+                    with tempfile.SpooledTemporaryFile(max_size=s_const.mebibyte * 128,
```

Comment:
128MiB feels kinda small. Should this be configurable?

Reply:
increased it to 512MiB and added test for rollover.

Reply:
made this large to line up with the axon; as well as a constant that we can change in tests via mock.